### PR TITLE
TAG: Add `GitlabEventStream` protos

### PR DESCRIPTION
**Pull request by tigrato**
This PR introduces the required boilerplate code to be able to import Gitlab structure into TAG

Part of https://github.com/gravitational/access-graph/issues/635

### CODE REVIEW

**Review by jakule — APPROVED** [review]

File: proto/accessgraph/v1alpha/gitlab.proto [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  string organization = 5;
+  // last_sign_in_at identifies the last sign in date.
+  google.protobuf.Timestamp last_sign_in_at = 6;
+  // can_create_group identifies if the user can create groups.
+  bool can_create_group = 7;
+  // can_create_project identifies if the user can create projects.
+  bool can_create_project = 8;
+  // two_factor_enabled identifies if the user has two factor authentication enabled.
+  bool two_factor_enabled = 9;
+  // identities repsents the idenity source for the user.
```
Comment (justinas) [inline proto/accessgraph/v1alpha/gitlab.proto:114]: s/repsents/represents

File: proto/accessgraph/v1alpha/gitlab.proto [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  repeated GitlabUserIdentity identities = 10;
+}
+
+// GitlabUserIdentity identifies the external identity of the user.
+message GitlabUserIdentity {
+  // provider identifies the identity provider.
+  string provider = 1;
+  // extern_uid identities the external uid of the identity.
```
Comment (justinas) [inline proto/accessgraph/v1alpha/gitlab.proto:122]: s/identities/identifies

File: proto/accessgraph/v1alpha/gitlab.proto [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  string path = 2;
+  // full_name is the group full name.
+  string full_name = 3;
+  // description is the group description.
+  string description = 4;
+}
+
+// GitlabProject represents a Gitlab project
+message GitlabProject {
+  // name is the repository name-
```
Comment (justinas) [inline proto/accessgraph/v1alpha/gitlab.proto:66]: s/-//

File: proto/accessgraph/v1alpha/access_graph_service.proto [hunk 4]
```diff
@@ -174,3 +178,20 @@ message AWSSyncOperation {}
...
+    // it's issued once Teleport finishes syncing all resources with the database.
+    GitlabSyncOperation sync = 1;
+    // upsert is a command to put a resource into the access graph or update it.
+    GitlabResourceList upsert = 2;
+    // delete is a command to delete a resource from the access graph when it's deleted from Teleport.
+    GitlabResourceList delete = 3;
+  }
+}
+
+// GitlabEventsStreamResponse asdasd
```
Comment (justinas) [inline proto/accessgraph/v1alpha/access_graph_service.proto:196]: 😄

File: proto/accessgraph/v1alpha/gitlab.proto [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+package accessgraph.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/accessgraph/v1alpha;accessgraphv1alpha";
+
+// GitlabSyncOperation is a request to sync Gitlab resources
+message GitlabSyncOperation {}
+
+// GitlabResourceList is a request to sync Gitlab resources
```
Comment (justinas) [inline proto/accessgraph/v1alpha/gitlab.proto:30]: Docstring for this is the same as for `GitlabSyncOperation`, but it probably should not be.

**Review by justinas — APPROVED** [review]
Just nits regarding the docstrings.

**Pull request merged**